docs/source/guide/FAQ.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: FAQ
+title: Troubleshoot Label Studio
```

Comment:
@smoreface Why FAQ is a not good variant?

Reply:
Most of the time FAQs aren't actually helpful to readers -- see [this article about why](https://content-guide.18f.gov/our-approach/structure-the-content/#dont-use-faqs). Most importantly for us, most of the content on this page isn't actually a question, even if it is frequently asked.

I plan to change this page into a troubleshooting page for errors (like CORS etc.) and then move the other helpful content (like the setting up pre-labeling) to dedicated pages or more relevant pages (like a second machine learning page about setting up and working with pre-labels, in addition to the existing page about setting up machine learning backends!)